$MirOS: ports/lang/tcl/8.4/patches/patch-unix_tcl_m4,v 1.1 2006/11/20 21:08:11 tg Exp $
--- unix/tcl.m4.orig	Wed Feb  6 16:38:03 2008
+++ unix/tcl.m4	Mon Feb 25 11:52:51 2008
@@ -1486,7 +1486,7 @@ dnl AC_CHECK_TOOL(AR, ar)
 	    UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.a'
 	    TCL_LIB_VERSIONS_OK=nodots
 	    ;;
-	OpenBSD-*)
+	OpenBSD-*|MirBSD-*)
 	    case `arch -s` in
 	    m88k|vax)
 		SHLIB_CFLAGS=""
@@ -2091,7 +2091,7 @@ dnl # preprocessing tests use only CPPFL
 		    ;;
 		IRIX*)
 		    ;;
-		NetBSD-*|FreeBSD-*|OpenBSD-*)
+		NetBSD-*|FreeBSD-*|OpenBSD-*|MirBSD-*)
 		    ;;
 		Darwin-*)
 		    ;;
